src/Platform/Microsoft.Testing.Platform/Messages/TestNodeProperties.cs

```diff
@@ -372,6 +374,15 @@ public record StandardOutputProperty(string StandardOutput) : IProperty;
 [Experimental("TPEXP", UrlFormat = "https://aka.ms/testingplatform/diagnostics#{0}")]
 public record StandardErrorProperty(string StandardError) : IProperty;
 
+/// <summary>
+/// Property that represents multiple artifacts/attachments to associate with a test node.
+/// </summary>
+/// <param name="SessionUid">The session UID.</param>
+/// <param name="FileInfo">The file information.</param>
+/// <param name="DisplayName">The display name.</param>
+/// <param name="Description">The description.</param>
+public record TestFileArtifactProperty(SessionUid SessionUid, FileInfo FileInfo, string DisplayName, string? Description = null) : IProperty;
```

Comment:
maybe for some time we could ship it as [Experimental("TPEXP", UrlFormat = "https://aka.ms/testingplatform/diagnostics#{0}")]

Reply:
Why not simply `FileArtifact`? 

cc: @Evangelink

Reply:
@MarcoRossignoli We always have `Property` suffix for properties, and there is already `FileArtifact` class (non-property that's used for TRX). So I assume you mean `FileArtifactProperty`? I'm fine with either name.

Reply:
`FileArtifactProperty` yep not bad imo